source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -0,0 +1,139 @@
+
+It would be possible to upload OVA files that were generated either by oVirt or by external sources. OVA files that were generated by external sources will be converted using virt-v2v, thus the supported external sources are limited to these whose OVA files can be converted by virt-v2v.  
+
+## Export as OVA
+Today, users are provided with two ways to migrate VMs and templates between environments. First, one can use an export domain. However, this approach is suitable only for migration between different oVirt environments. Moreover, there is a plan to drop the support for export domains. Second, one can detach a data domain and attach it to another setup. This way is tailored to disaster recovery and less fits the migration flow due to the side effects in detaching a data domain.  
+
+This feature provides an alternative way for a migration between environments by generating an OVA from an existing VM or template. By encapsulating all VM/template data (optionally compressed) in a single file, we achieve an improved portability. By being aligned with the common OVA specification (rather than our unique structure of a storage domain), we achieve 
+better compatibility with other platforms.  
+
+The OVA file would be created at a given path within a particular host (this capability is targeted for extracting the VM/template into an external storage used for backup/migration). In addition, it will include neither snapshots nor the template that the VM may be based on.  
+
+## Enhance Import OVA
+This feature enhances the current support provided for import OVA files in two ways. First, it would be possible to specify a path to OVA that was generated by oVirt. Import of OVA that was generated by oVirt does not involve conversion and thus, should be faster than importing OVA files generated by others.  
+
+Second, in order to provide better support for typical backup/restore and migration processes we enable listing all the OVA files in a given directory.  
+
+## Download OVA
+This feature will leverage the implementation of the planned download-image to provide similar functionality for OVA files.  
+
+# Design
+This section details the high-level design of the different aspects described in the previous section.
+
+## Upload OVA
+In principle, uploading an OVA is similar to uploading an image. However, it is more challenging because OVA is a more complex resource than an image.  
+
+The upload process would be implemented differently depending on whether the OVA was generated by oVirt or by other source (and thus requires conversion). In both cases, the user will be able to select a file from within a browser and be presented with the VM configuration that will allow him to configure the parameters for the import process:
+
+1. The user selects an OVA file from within the browser.
+2. The front-end sends the OVF to the back-end.
```

Comment:
we do not need OVA exports on source oVirt hosts, we should not require kludges as we do today with OVA import(manual mount, iso domain workaround).
imageio-proxy sitting by default on engine host and deployable anywhere else is a better "target" since it can write to more suitable locations (e.g. it can see the other oVirt environment, or user's NFS archive directory)